### PR TITLE
Release changelog for 2.8.5

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@ubuntu.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+  * Update the busybox test to use the snap data path. (LP: #1574901)
+  * Remove the integration tests coverage. (#488) (LP: #1575383)
+
+  [ Sergio Schvezov ]
+  * Don't clean target before extracting npm (#489) (LP: #1575876)
+  * Don't delete the nodejs download on build. (#490) (LP: #1575882)
+
+  [ Vincent Ladeuil ]
+  * Make upload more robust by ignoring suprious errors while polling the
+    scan status. (#480) (LP: #1572963)
```
Comment (kyrofa) [inline debian/changelog]: Should we fix the typo here?

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+  * autopkgtests: run the tests using the installed package (#464)
+    (LP: #1570992)
+  * Remove --allow-unauthenticated from examples tests (#482) (LP: #1573211)
+  * Update the assertion of the example install (#483) (LP: #1573243)
+  * Examples tests: Update the path to the snaps binaries (#484) (LP: #1573349)
+  * Examples tests: use systemctl instead of the removed snap service (#485)
+    (LP: #1573697)
+  * Update the mosquitto example SNAP_USER_DATA path. (#486) (LP: #1574857)
+  * Update the busybox test to use the snap data path. (LP: #1574901)
+  * Remove the integration tests coverage. (#488) (LP: #1575383)
```
Comment (elopio) [inline debian/changelog]: I won this time!

File: debian/changelog (hunk 1 above)
Comment (sergiusens) [inline debian/changelog]: yes!

**Comment by elopio** [comment]
:100:

File: debian/changelog (hunk 2 above)
Comment (kyrofa) [inline debian/changelog]: Man you sure did. Awesome changes, too! You're the MVP of this release.

**Comment by kyrofa** [comment]
Awesome :+1:

File: debian/changelog (hunk 2 above)
Comment (elopio) [inline debian/changelog]: The trick is to do one line per PR ^,^

**Comment by elopio** [comment]
restest this please

**Pull request merged**